### DISCUSSION TITLE
Deploying using AWS Lambda Python 3.10

### DISCUSSION THREAD

**Discussion by gadi-playstream-gg** [openai/whisper#1376] [Q&A]
I'm gettting errors regarding missing "ffmpeg" file when trying to transcribe a WAV file.
I'm new to python and docker files.
Anyone deployed it using Python 3.10 and can share docker file and / or instructions how can this be done?

Thanks!

**Comment by phineas-pta** [comment]
search for "AWS" you'll see other threads

**Reply by gadi-playstream-gg** [reply]
I did , couldn't really find the answer there. I'm using Windows.
here's the default docker file:

FROM public.ecr.aws/lambda/python:3.10
COPY app.py requirements.txt ./
RUN python3.10 -m pip install -r requirements.txt -t .
CMD ["app.lambda_handler"]

Do i need to add / change something in the docker file? do I need to add ffmpeg to the requirements.txt?

**Reply by phineas-pta** [reply]
you can try adding `apt-get install ffmpeg`

**Reply by rvryan67** [reply]
Try adding following to dockerfile

# install helpers
RUN yum -y install tar xz
# install ffmpeg
RUN curl https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz > /tmp/ffmpeg-release.tar.xz && tar xvf /tmp/ffmpeg-release.tar.xz -C /opt && mv /opt/ffmpeg-* /opt/ffmpeg && cd /opt/ffmpeg && mv model /usr/local/share && mv ffmpeg ffprobe qt-faststart /usr/local/bin && rm /tmp/ffmpeg-release.tar.xz
FROM  public.ecr.aws/lambda/python:3.9 as app
COPY --from=builder /opt/ffmpeg /opt/ffmpeg/
COPY --from=builder /usr/local/share/model /usr/local/share/
COPY --from=builder /usr/local/bin/ff* /usr/local/bin/
COPY --from=builder /usr/local/bin/qt-* /usr/local/bin/
